src/moments.jl

```diff
@@ -30,23 +32,32 @@ whereas it's `length(x)-1` in `Base.var`. The impact is that this is not a
 weighted estimate of the population variance based on the sample; it's the weighted
 variance of the sample.
 """
-function Base.var(v::RealArray, wv::WeightVec; mean=nothing)
-    mean == 0 ? Base.varm(v, wv, 0) :
-    mean == nothing ? varm(v, wv, Base.mean(v, wv)) :
-    varm(v, wv, mean)
+function Base.var(v::RealArray, wv::AbstractWeights; mean=nothing, corrected=true)
+    mean == 0 ? varm(v, wv, 0; corrected=corrected) :
+    mean == nothing ? varm(v, wv, Base.mean(v, wv); corrected=corrected) :
+    varm(v, wv, mean; corrected=corrected)
 end
 
 ## var along dim
 
-Base.varm!(R::AbstractArray, A::RealArray, wv::WeightVec, M::RealArray, dim::Int) =
-    scale!(_wsum_centralize!(R, @functorize(abs2), A, values(wv), M, dim, true), inv(sum(wv)))
+function Base.varm!(R::AbstractArray, A::RealArray, wv::AbstractWeights, M::RealArray, dim::Int; corrected=true)
+    scale!(
```

Comment:
Everyone has a different preferred style; it's mostly about being consistent within a particular package. That said, I'd love for the style guide in the manual to be more specific about things.